make test failed:    can't load package: named files must be .go files:xxx

<!-- Please only use this template for submitting reports about failing tests in KubeEdge CI jobs -->

**Which jobs are failing**:
unit test
**Which test(s) are failing**:
make test
**Since when has it been failing**:
make test
**Reason for failure**:
when I add a unit test. Has nothing to do with the following path 
```
[root@centos-kubeedge kubeedge]# make test
hack/make-rules/clean.sh
hack/make-rules/test.sh 
running tests cases 
can't load package: named files must be .go files: ./cloud/pkg/cloudhub/common/model
make: *** [test] Error 1
```
**Anything else we need to know**:
I find the the reason in  hack/lib/golang.sh  kubeedge::golang::get_keadm_test_dirs()
when I try  to perform this step, not only get the directory, but also get the go file
```
[root@centos-kubeedge kubeedge]# find -L ./keadm \
    -name '*_test.go' -print | xargs -0n1 dirname | uniq
./keadm/cmd/keadm/app/cmd/debug/collect_test.go
./keadm/cmd/keadm/app/cmd/util
```

> The issue title is so confused util i see the details.
> 
> I think it is not a failing-test, it is related to how you add a test case or test file indeed.
> 
> So, you shoulmay fille how you add test case/file, rather than writing `make test` in each description.ok I only add

ok, I only add one unit test, in `./keadm/cmd/keadm/app/cmd/debug/collect_test.go`
```
package debug

import (
	types1 "github.com/kubeedge/kubeedge/keadm/cmd/keadm/app/cmd/common"
	"testing"
)

func TestVerificationParameters(t *testing.T) {
...
}
```

when I make test it is failed, i try to remove this unit test. it is successful

I print  `${KUBEEDGE_ALL_TESTCASES[@]}` in kubeedge::golang::run_test, and find this unit test can not get dir and get go file. 
```
[root@centos-kubeedge kubeedge]# make test
hack/make-rules/clean.sh
hack/make-rules/test.sh 
running tests cases 
./cloud/pkg/cloudhub/common/model ./cloud/pkg/edgecontroller/manager ./edge/pkg/devicetwin ./edge/pkg/devicetwin/dtclient ./edge/pkg/devicetwin/dtcommon ./edge/pkg/devicetwin/dtcontext ./edge/pkg/devicetwin/dtmanager ./edge/pkg/devicetwin/dtmodule ./edge/pkg/devicetwin/dttype ./edge/pkg/devicetwin ./edge/pkg/edgehub/clients/wsclient ./edge/pkg/edgehub/common/http ./edge/pkg/edgehub ./edge/pkg/eventbus/common/util ./edge/pkg/metamanager/dao ./edge/pkg/metamanager ./edge/pkg/servicebus/util ./keadm/cmd/keadm/app/cmd/debug/collect_test.go
can't load package: named files must be .go files: ./cloud/pkg/cloudhub/common/model
make: *** [test] Error 1
```

Finally positioned  cmd `find -L ./keadm \
    -name '*_test.go' -print | xargs -0n1 dirname | uniq` in `kubeedge::golang::get_keadm_test_dirs()`
I am not sure about the usage of `xarg -0`

--- a/hack/lib/golang.sh
+++ b/hack/lib/golang.sh
@@ -371,8 +371,9 @@ kubeedge::golang::get_cloud_test_dirs() {
 kubeedge::golang::get_keadm_test_dirs() {
     cd ${KUBEEDGE_ROOT}
     findDirs=$(find -L ./keadm \
-	    -name '*_test.go' -print | xargs -0n1 dirname | uniq)
-    echo "${findDirs}"
+	    -name '*_test.go' -print | xargs -n1 dirname | uniq)
+    dirArray=(${findDirs// /})
+    echo "${dirArray[@]}"
 }
 
 kubeedge::golang::get_edge_test_dirs() {

